### ISSUE TITLE
Parameter fails to be named correctly

### ISSUE THREAD

**Issue by PeqNP** [Nordstrom/bluffalo#5]
The method signature for the following method:

```
internal func selectPrimarySize(_ groupIndex: Int, _ optionIndex: Int) {
   ...
}
```

Produces the following fake method signature.

```
public static func selectPrimarySize(_ groupIndex: Int, _ groupIndex: Int) -> ProductDetailsKitchenStub<Any> {
    return ProductDetailsKitchenStub<Any>(method: .selectPrimarySize_groupIndex_optionIndex(groupIndex, optionIndex))
}
```

Expected: The second parameter of this method signature should be `optionIndex`.
Actual: The second parameter is the same name as the first paramter, `groupIndex`.

**Comment by PeqNP** [comment]
This is fixed in Pull Request #7

**Comment by PeqNP** [comment]
This has been fixed and merged into master.
